akvo/rsr/spa/app/modules/results-admin/components/AuditTrailModal.jsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react'
+import {
+  Row,
+  Col,
+  Collapse,
+  Modal,
+  Skeleton,
+  Typography,
+  Descriptions,
+} from 'antd'
+import moment from 'moment'
+import { Icon } from '../../../components'
+import { AuditTrail } from '../../../components/AuditTrail'
+import { setNumberFormat } from '../../../utils/misc'
+import { StatusPeriod } from '../../../components/StatusPeriod'
+
+const { Title, Text } = Typography
+const { Panel } = Collapse
+
+const AuditTrailModal = ({
+  scores = [],
+  results = [],
+  item,
+  visible,
+  onClose,
+}) => {
+  const { fetched, period, indicator } = item || {}
+  const defaultActiveKey = results.map((r) => r.id).slice(0, 1)
+  return (
+    <Modal
+      centered
+      closable
+      visible={visible}
+      width={710}
+      onCancel={onClose}
+      cancelText="Close"
+      title={(
+        <div
+          style={{
+            display: 'flex',
+            gap: 8,
+            alignItems: 'center'
+          }}
+        >
+          <Icon type="clock.history" />
+          <Title level={4} strong>Audit trail</Title>
+        </div>
+      )}
+    >
+      <div className="period-caption">
+        {`${moment(period?.periodStart, 'DD/MM/YYYY').format('DD MMM YYYY')} - ${moment(period?.periodEnd, 'DD/MM/YYYY').format('DD MMM YYYY')}`}
```

Comment:
It would be nice to create variable and assign values and render variable. If later on we convert it to component it would be easier to just pass the variable instead